style.css

```diff
@@ -0,0 +1,232 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;400;500;600;700;800&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
```

Comment:
[ ] kindly change your `font-family`, the one used on the figma design is` Inter`.

![font](https://user-images.githubusercontent.com/62156607/198273248-0df886e1-51d6-4334-9d3d-53d2b744a047.PNG)

Reply:
@nedjwab the template i chose to work on has a font of poppin